### ISSUE TITLE
Account Section locked

### ISSUE THREAD

**Issue by K420K** [goatfungus/NMSSaveEditor#898]
First of all I've been using this awesome program for a while...but I'm noticing that with the ps4 save everything works except the account section which I can't access...any solutions?

**Comment by Devilin-Pixy** [comment]
@K420K You'd have to blame Save Wizard for that, as it is not decrypting the savedata00 (account data) for as far as I am aware.

**Comment by K420K** [comment]
I don't think after all this time it happens...anyway I also use another program to extract files(Apollo) if the developer would accept my file he could implement that too...which by the way are the same as that sw

**Comment by Devilin-Pixy** [comment]
Ah ok, I am not sure the Apollo Save Tool (https://github.com/bucanero/apollo-ps4) is able to decrypt the account data file (savedata00). I personally do not own a PS4 to verify anything. I can however imagine it being useful if @goatfungus would be able to implement support for it.

**Comment by K420K** [comment]
Yes Apollo decrypts it,however once I put it on the nms app to edit it does not read it...yet they are in the usual "savedataxx.hg" format

**Comment by 1stCawthorne** [comment]
it does the same from Xbox 1 to PC saves
